flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,256 @@
+                                continue
+                            else:
+                                for episode in season['episodes']:
+                                    tempSeries_id = 'S' + str(season["seasonNumber"]).zfill(2) + 'E' + str(episode["episodeNumber"]).zfill(2)
+                                    entry = Entry(title=generate_title(request, config.get('strip_year')) + ' ' + tempSeries_id + ' ' + episode['title'],
+                                                  url=episode.get('url'),
+                                                  series_name=generate_title(request, config.get('strip_year')),
+                                                  series_season=season["seasonNumber"],
+                                                  series_episode=episode["episodeNumber"],
+                                                  series_id=tempSeries_id,
+                                                  tvdb_id=request.get('tvDbId'),
+                                                  imdb_id=request.get('imdbId'),
+                                                  ombi_request_id=request.get('id'),
+                                                  ombi_childrequest_id=childRequest["id"],
+                                                  ombi_season_id=season["id"],
+                                                  ombi_episode_id=episode["id"],
+                                                  ombi_approved=episode.get('approved'),
+                                                  ombi_available=episode.get('available'),
+                                                  ombi_requested=episode.get('requested'))
+                                    if entry.isvalid():
+                                        log.debug('Valid entry %s', entry)
+                                        if config.get('only_approved') and not entry.get('ombi_approved'):
+                                            log.verbose('Request not approved skipping: %s', entry.get('title'))
+                                        else:
+                                            if not config.get('include_available') and entry.get('ombi_available'):
+                                                log.verbose('Request already available skipping: %s', entry.get('title'))
+                                            else:
+                                                entries.append(entry)
+                                    else:
+                                        log.error('Invalid entry created? %s', entry)
+                                        continue
+            return entries
+ 
+class OmbiSet(MutableSet):
+    supported_ids = ['imdb_id', 'tmdb_id', 'ombi_id']
+    schema = {
+        'type': 'object',
+        'properties': {
+            'base_url': {'type': 'string'},
+            'port': {'type': 'number', 'default': 3579},
+            'api_key': {'type': 'string'},
```

Comment:
Consider adding username/password support. I started an ombi_list plugin and I have implemented both in a simple function. I can post it here later if you want.

Reply:
Went with the URI api key for easy and similarity to other plugins. Also integration is the only reason Ombi generates a api_key so I am not sure why username and password would be better. There is a another api_key method using post methods as well I believe.

Reply:
The api key gives the user admin rights. Username/password uses the built-in role administration.

Reply:
Hmm I will look into it.

Reply:
User name and password support has been added.